Azure/terraform/README.md

```diff
@@ -6,16 +6,23 @@ These terraform scripts deploy [FME Flow (Distributed deployment, Windows)](http
 
 1. Review the `variables.tf` file. This file contains all variables for the deployment. Most of the variables have default values assigned, but can be changed in the `.tf` file or overridden by using the `-var` flag with the `terraform apply` command. You will be prompted for any variable that does not have a default after running the `terraform apply` command.
-2. Run `terraform apply` in your console from the directory that that holds the `main.tf` and `variables.tf` file and provide any variables you are prompted for.
-3. Review the deployment plan. If the terraform script and the provided variables validated successfully the deployment plan will be output in the consoled for you to review. Additionally you will be prompted wether you want to go ahead with the deployment. If everything looks ok, go ahead with `yes`. The deployment will now provision and configure all necessary Azure resources and start up FME Flow. This will take about 10 - 20 minutes.
-4. In this quickstart example the terraform statefile `terraform.tfsate` will be created on on your local machine, so you can review the current state of your deployment and test the deployment. For any productive deployments it is highly recommended to not store the state file locally but in a remote location. This makes sure you can collaborate on the state and any sensitive data contained in the state file will only be accessible to authorized users. To use Azure storage as a backend for your statefile follow this documentation: [Azure storage terraform backend](https://www.terraform.io/language/settings/backends/azurerm)
+2. If you want to deploy into a existing resource group in Azure, you will need to make the following edits to `main.tf`: 
+   - replace `resource azurerm_resource_group "fme_flow"` with `data "azurerm_resource_group" "fme_flow"`
+   - remove or comment out the `location` and `tags` lines from this same block
+   - prepend `data.` in front of all occurrences of `azurerm_resource_group.fme_flow` in the template
+3. If you created your own FME images via packer, you will need to edit the `main.tf` files under `modules/vmss/vmss_core` and `vmss_engine` to include the custom `source_image_id` and remove the references to the marketplace images (see comment above `source_image_reference` line).
+4. Run `terraform apply` in your console from the directory that that holds the `main.tf` and `variables.tf` file and provide any variables you are prompted for.
+5. Review the deployment plan. If the terraform script and the provided variables validated successfully the deployment plan will be output in the consoled for you to review. Additionally you will be prompted wether you want to go ahead with the deployment. If everything looks ok, go ahead with `yes`. The deployment will now provision and configure all necessary Azure resources and start up FME Flow. This will take at least 10 - 20 minutes.
```

Comment:
```suggestion
5. Review the deployment plan. If the terraform script and the provided variables are validated successfully the deployment plan will be output in the console for you to review. Additionally you will be prompted whether you want to go ahead with the deployment. If everything looks ok, go ahead with `yes`. The deployment will now provision and configure all necessary Azure resources and start up FME Flow. This will take at least 10 - 20 minutes.
```